<?xml version="1.0" ?>
<sdf version="1.3">
  <model name="fcsc_shelf">
    <static>true</static>

    <link name="base">
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_base.dae</uri>
          </mesh>
        </geometry>
      </visual>
      <collision name="simple_collision">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_base.dae</uri>
          </mesh>
        </geometry>
      </collision>
    </link>

    <link name="box">
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_box.dae</uri>
          </mesh>
        </geometry>
      </visual>
      <collision name="simple_collision">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_box.dae</uri>
          </mesh>
        </geometry>
      </collision>
    </link>

    <link name="board_org">
      <pose>0 0 0.31  0 0 0</pose>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_board.dae</uri>
          </mesh>
        </geometry>
      </visual>
      <collision name="simple_collision">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_board.dae</uri>
          </mesh>
        </geometry>
      </collision>
    </link>

    <link name="board0">
<pose>0.000000 0.000000 0.800000  0 0 0</pose>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_board.dae</uri>
          </mesh>
        </geometry>
      </visual>
      <collision name="simple_collision">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_board.dae</uri>
          </mesh>
        </geometry>
      </collision>
    </link>

    <link name="board1">
<pose>0.000000 0.000000 1.030000  0 0 0</pose>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_board.dae</uri>
          </mesh>
        </geometry>
      </visual>
      <collision name="simple_collision">
        <geometry>
          <mesh>
            <uri>model://fcsc_shelf/meshes/shelf_board.dae</uri>
          </mesh>
        </geometry>
      </collision>
    </link>

  </model>
</sdf>
